### DISCUSSION TITLE
Add conversion to UTF-16

### DISCUSSION THREAD

**Discussion by tayopi** [character-map-uwp/Character-Map-UWP#267] [Ideas]
In Nerd Fonts v3.0.0 the Material Design Icons were dropped from the F500... code points and moved to the F0001 code point. As a result of these new five digit codes it is sometimes necessary to convert the HEX value to UTF-16.

**Example**
The following information is for the altimeter glyph:
Unicode Hex = U+F05D7
UTF-16 code = \udb81\uddd7

**Comment by JohnnyWestlake** [comment]
#270

**Reply by tayopi** [reply]
Thank you!
